README.rst

```diff
@@ -211,48 +211,49 @@ the ``--command`` argument and execute the ``cov-run-desktop`` through
     python -m mlx.warnings --coverity --command <commandforcoverity>
 
 
-We utilize `cov-run-desktop` in the following manner, where the output is saved in `coverity.log`:
+The command below demonstrates how we utilize `cov-run-desktop`:
 
 .. code-block:: bash
 
-    cov-run-desktop --text-output-style=oneline --exit1-if-defects false --triage-attribute-regex "classification" ".*" <coverity_files> | tee coverity.log
+    cov-run-desktop --text-output-style=oneline --exit1-if-defects false --triage-attribute-regex "classification" ".*" <coverity_files> | tee raw_defects.log
 
-Subsequently, we process the `coverity.log` file with the mlx-warnings plugin.
-The plugin uses a configuration file (`warnings_coverity.yml`) and produces two outputs:
-a text file (`warnings_coverity.txt`) and a code quality JSON file (`coverity_code_quality.json`).
+Then, the mlx-warnings plugin processes the output log file, `raw_defects.log`, based on the configuration file
+`warnings_coverity.yml` to produces three outputs:
+
+- A text file called `warnings_coverity.txt`, which contains all Coverity defe.
+- A code quality JSON file `coverity_code_quality.json`.
+- A return code equal to the amount of violations, i.e. the amount of Coverity defects that exceeds the configured
+  limit(s). The value is 0 if the amount of Coverity defects is within limits. We use this return code to determine
+  whether our CI job passes or fails.
 
 .. code-block:: bash
 
-    mlx-warnings --config warnings_coverity.yml -o warnings_coverity.txt -C coverity_code_quality.json coverity.log
+    mlx-warnings --config config.yml --output counted_defects.txt --code-quality report.json raw_defects.log
 
-This is an example of the configuration file:
+Below is an example configuration for the Coverity checker:
 
 .. code-block:: yaml
 
-    sphinx:
-        enabled: false
-    doxygen:
-        enabled: false
-    junit:
-        enabled: false
-    xmlrunner:
-        enabled: false
     coverity:
         enabled: true
-        intentional:
-            max: -1
-        bug:
-            max: 0
+        unclassified:
+          max: 0
         pending:
-            max: 0
+          max: 0
         false_positive:
-            max: -1
-    robot:
-        enabled: false
-    polyspace:
-        enabled: false
-
-For each classification, a minimum and maximum can be given.
+          max: -1
+        intentional:
+          max: -1
+        bug:
+          min: 2
+          max: 2
+
+As you can see, we have configured limits for 5 out of 5 Coverity Classifications. You can configure a minimum and a
+maximum limit for the number of allowed Coverity defects the belong the the Classification.
+The default value for both limits is 0.
+A value of `-1` for `max` corresponds to effectively no limit (an infinite amount).
+If one or more Classifications are not present in your configuration, the corresponding defects will be ignored
+completely.
```

Comment:
Is this last sentence true? It's [what I requested a year ago](https://github.com/melexis/warnings-plugin/issues/128#issue-1947095155), but I don't remember if we decided on an alternative implementation in the end.

Reply:
This is not true, at least for now. I can change that.

Reply:
Let's keep the implementation as is, since it accomodates the use of the `--coverity` input argument, without the `--config`. I fixed this sentence in 7a80fbcac5488cc285138bebd30317a095403141